mmv1/third_party/terraform/services/bigquerydatatransfer/resource_bigquery_data_transfer_config_test.go

```diff
@@ -616,6 +616,27 @@ func testAccBigqueryDataTransferConfig_salesforce_basic(t *testing.T) {
 	})
 }
 
+func testAccBigqueryDataTransferConfig_pub_sub_basic(t *testing.T) {
```

Comment:
```suggestion
func TestAccBigqueryDataTransferConfig_pub_sub_basic(t *testing.T) {
```

Test names need to be capitalized so that they are exported and run.